### PR TITLE
[docs] Platform FAQ fixes

### LINKED ISSUE #7267: [Docs] Yugaware should be Yugabyte Platform
Starting in v2.2, the name changed. Update all references in v2.2+, stable, and latest...

Helm charts still use the Yugaware name, probably also package and other file names.

### PR DESCRIPTION
**Pull request by ddhodge**
Link fixes, remove references to Yugaware
This also probably resolves #7267 

@netlify /latest/faq/yugabyte-platform/

### CODE REVIEW

File: docs/content/latest/faq/yugabyte-platform.md [hunk 1]
```diff
@@ -14,92 +14,85 @@ isTocNested: false
...
+- Ability to connect from the Yugabyte Platform host to all YugabyteDB data nodes via SSH.
 
-- Following ports should be open on the YugaWare host: `8800` (replicated ui), `80` (http for YugabyteDB Admin Console), `22` (ssh)
-- Attached disk storage (such as persistent EBS volumes on AWS): 100 GB minimum
-- A Yugabyte Platform license file from [Yugabyte](https://www.yugabyte.com/platform/#request-trial-form)
-- Ability to connect from the Yugabyte Platform host to all YugabyteDB data nodes. If this is not set up, [setup passwordless ssh](#step-5-troubleshoot-yugaware).
+For a complete list of prerequisites, refer to [Prerequisites](../../yugabyte-platform/install-yugabyte-platform/prerequisites/).
 
-## What are the OS requirements and permissions to run the YugabyteDB data nodes?
+## What are the OS requirements and permissions to run YugabyteDB data nodes?
```
Comment (aishwarya24) [inline docs/content/latest/faq/yugabyte-platform.md:56]:
```suggestion
## What are the OS requirements to run YugabyteDB data nodes?
```
There seems to be no information about _permissions_ under the Deployment checklist

File: docs/content/latest/faq/yugabyte-platform.md [hunk 2]
```diff
@@ -14,92 +14,85 @@ isTocNested: false
...
-
-An _airgapped_ host has no path to inbound or outbound Internet traffic at all. For such hosts, the installation is performed as a `sudo` user.
-
-**Additional requirements**
+Yugabyte Platform also requires the following:
 
-For airgapped hosts, a supported version of docker-engine (currently 1.7.1 to 17.03.1-ce). If you do not have docker-engine installed, follow the instructions [here](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/) to first install docker-engine.
+- Connectivity to the Internet, either directly or via an HTTP proxy.
+- Ability to install and configure [docker-engine](https://docs.docker.com/engine/).
+- Ability to install and configure [Replicated](https://www.replicated.com/), which is a containerized application itself and needs to pull containers from its own Replicated.com container registry.
```
Comment (aishwarya24) [inline docs/content/latest/faq/yugabyte-platform.md:47]: Maybe a good idea to point to the [Replicated.com container registry](https://help.replicated.com/docs/native/getting-started/docker-registries/). I am guessing this is the one.

File: docs/content/latest/faq/yugabyte-platform.md [hunk 3]
```diff
@@ -14,92 +14,85 @@ isTocNested: false
...
 
-The data node software is packaged into the Yugabyte Platform application. Yugabyte Platform distributes and installs the data node software on the hosts identified to run the data nodes. Since it's already packaged into existing artifacts, the data node does not require any Internet connectivity.
+Installation of the admin console starts with installing Replicated on a Linux host. Replicated installs the [docker-engine](https://docs.docker.com/engine/), the Docker container runtime, and then pulls its own container images from the Replicated.com container registry. Yugabyte Platform then becomes a managed application of Replicated, which starts by pulling the Yugabyte Platform (`yugaware`) container images from Quay.io for the very first time.
```
Comment (aishwarya24) [inline docs/content/latest/faq/yugabyte-platform.md:64]: I mentioned earlier , about linking to the [Replicated.com container registry](https://help.replicated.com/docs/native/getting-started/docker-registries/). Just adding it again.

File: docs/content/latest/faq/yugabyte-platform.md [hunk 4]
```diff
@@ -14,92 +14,85 @@ isTocNested: false
...
-**Permissions necessary for an airgapped host**
-
-An _airgapped_ host has no path to inbound or outbound Internet traffic at all. For such hosts, the installation is performed as a `sudo` user.
-
-**Additional requirements**
+Yugabyte Platform also requires the following:
 
-For airgapped hosts, a supported version of docker-engine (currently 1.7.1 to 17.03.1-ce). If you do not have docker-engine installed, follow the instructions [here](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/) to first install docker-engine.
+- Connectivity to the Internet, either directly or via an HTTP proxy.
+- Ability to install and configure [docker-engine](https://docs.docker.com/engine/).
```
Comment (aishwarya24) [inline docs/content/latest/faq/yugabyte-platform.md:46]: [docker-engine](https://docs.docker.com/engine/install/) - point over to the install page directly ?

File: docs/content/latest/releases/earlier-releases/v2.0.9.md [hunk 5]
```diff
@@ -70,16 +67,15 @@ docker pull yugabytedb/yugabyte:2.0.9.0-b13
...
 
 ## Platform
+
 * [YW] Change Intro dialog to only appear on login. Add checkbox in Intro dialog for permanently hiding modal. [#3138](https://github.com/yugabyte/yugabyte-db/issues/3138)
 * [Platform] Fix console errors in UI code [#2790](https://github.com/yugabyte/yugabyte-db/issues/2790)
 * [YW] Fix backup Universe from changing to undefined url
 * [Platform] Task history not reflecting when a encryption is enabled/disabled/rotated [#2929](https://github.com/yugabyte/yugabyte-db/issues/2929)
 * Add per node metrics for health. [#2894](https://github.com/yugabyte/yugabyte-db/issues/2894)
-* Implement RBAC in Yugaware. [#2793](https://github.com/yugabyte/yugabyte-db/issues/2793)
+* Implement RBAC in YugaWare. [#2793](https://github.com/yugabyte/yugabyte-db/issues/2793)
```
Comment (aishwarya24) [inline docs/content/latest/releases/earlier-releases/v2.0.9.md:76]:
```suggestion
* Implement RBAC in Yugabyte Platform(YugaWare). [#2793](https://github.com/yugabyte/yugabyte-db/issues/2793)
```
Just suggesting the same way based on how you have replaced it at some of the other places.

File: docs/content/latest/faq/yugabyte-platform.md (hunk 4 above)
Comment (ddhodge) [reply docs/content/latest/faq/yugabyte-platform.md:46]: Linked overview page has links to install along with other info - I think it's good as is

File: docs/content/latest/releases/earlier-releases/v2.0.9.md (hunk 5 above)
Comment (ddhodge) [reply docs/content/latest/releases/earlier-releases/v2.0.9.md:76]: As at that release (2.0.9) product was still called YugaWare so I left it

**Comment by aishwarya24** [comment]
LGTM.

**Pull request merged**